src/misc/ssEconomy.lua

```diff
@@ -218,9 +228,24 @@ function ssEconomy:calculateLoanInterestRate()
     g_currentMission.missionStats.loanAnnualInterestRate = seasonsYearInterest
 end
 
+function ssEconomy:aiLoad(savegame)
+
+	-- After loading the aiVehicle, convert the vanilla pricePerMS to Seasons values (2000 to 1650 per hour), and store in a new variable, so pricePerMS can be altered without losing the original value.
+	self.originalPricePerMS = self.pricePerMS * g_seasons.economy.aiVanillaToSeasonsMultiplier;
+end
+
 function ssEconomy.aiUpdateTick(self, superFunc, dt)
-    if self:getIsActive() and self.pricePerMS ~= 0 then
-        self.pricePerMS = ssUtil.isWorkHours() and g_seasons.economy.aiPricePerMSWork or g_seasons.economy.aiPricePerMSOverwork
+    if self:getIsActive() then
```

Comment:
As stated above, I wanted to make sure that we're always working with the seasons pricePerHour. It's very easy to forget one of those /2000's. Tom Scott made a [great video](https://www.youtube.com/watch?v=QPZ0pIK_wsc) about this recently.